src/main/java/com/se/homeworktwo/Sym.java

```diff
@@ -50,7 +55,8 @@ String mid() {
     double div(){
         double e = 0;
         for (String key : _has.keySet()) {
-           e -= unit_entropy(_has.get(key)/n);
+           double freq = _has.get(key);
+           e -= unit_entropy(freq/n);
```

Comment:
Why do we need this?

Reply:
We need to typecast to double  so that freq/n does not evaluate to 0 and divison by zero is prevented.